docs/DeveloperGuide.md

```diff
@@ -782,34 +782,38 @@ FitBook's team size is 5.
 
 1. **Allow for more flexible weight management**
 
-The current implementation of tracking clients' weights only allows users to add, modify or delete the latest weight value at the current date and time. We plan to make this feature more flexible by allowing users to add/modify a client's weight at a specified date and time of their choice.
+    The current implementation of tracking clients' weights only allows users to add, modify or delete the latest weight value at the current date and time. We plan to make this feature more flexible by allowing users to add/modify a client's weight at a specified date and time of their choice.
 
-2. **Provide more specific error messages**
+1. **Provide more specific error messages**
```

Comment:
Does changing all the numbering to 1s here work? It does not seem to be rendering properly on the Jekyll website

![image](https://github.com/AY2324S2-CS2103T-T17-3/tp/assets/44436551/811d3854-1154-40d0-bc59-7df0855ad6e4)

Reply:
Yep I noticed that too, it works now because I indented the text under each point.